web/src/components/shared/storageDefinitions.ts

```diff
@@ -14,127 +15,149 @@ export const storageDefinitions: StorageDefinition[] = [
+    riskScore: 1,
   },
   {
     name: "Courier service (secured)",
     value: "courier_service_secured",
     definition: "Including tamper-evident packaging and Tracked and signed for.",
+    riskScore: 5,
   },
   {
     name: "Data Controller (external to UCL)",
     value: "data_controller_external",
     definition:
       "A Data Controller is an organisation which has determined its own reasons for processing personal data and has an established lawful basis for doing so.",
+    riskScore: 7,
   },
   {
     name: "Data entry",
     value: "data_entry",
     definition: "Authentication required to access, session secured with end to end encryption (HTTPS or VPN)",
+    riskScore: 3,
   },
   {
     name: "Data Safe Haven",
     value: "data_safe_haven",
     definition:
       "If sending files via the Data Safe Haven File Transfer Portal to an email recipient, users must use a different password delivery mechanism to the email address used. Storage locations used: (1) Torrington Place, 1-19 Torrington Pl, London WC1E 7HB, (2) Slough, VIRTUS LONDON4, 14 Liverpool Road, Slough, Berkshire SL1 4QZ, (3) AWS EU-West-2, London.",
+    riskScore: 1,
   },
   {
     name: "Data Safe Haven applications",
     value: "data_safe_haven_applications",
     definition: "Must use a separate password delivery mechanism to email.",
+    riskScore: 1,
   },
   {
     name: "Data Safe Haven file transfer portal",
     value: "data_safe_haven_file_transfer",
     definition:
       "If sending files via the Data Safe Haven File Transfer Portal to an email recipient, users must use a different password delivery mechanism to the email address used.",
+    riskScore: 1,
   },
   {
     name: "Desktop@UCL staff computer",
     value: "desktop_ucl_staff",
     definition:
       "UCL Information Services Division will issue on request a standard desktop installation called Desktop@UCL which includes encryption of the device's storage and anti-malware. However, some departments have not migrated to the Desktop@UCL service and will issue devices which are not necessarily as secure by default.",
+    riskScore: 2,
   },
   {
     name: "DfE network",
     value: "dfe_network",
     definition:
       "The Department for Education's Data Sharing Agreement makes these data highly confidential by default. The contract must be adhered to. Advice is best sought from IG Advisory (slms.pid@ucl.ac.uk) before entering these agreements.",
+    riskScore: 1,
   },
   {
     name: "Email",
     value: "email",
     definition:
       "Consult with IG Advisory (slms.pid@ucl.ac.uk) if you need to use email to send or receive research participants' personal data.",
+    riskScore: 7,
   },
   {
     name: "Fax",
     value: "fax",
     definition:
       "Consult with IG Advisory (slms.pid@ucl.ac.uk) if you need to use fax to send or receive research participants' personal data.",
+    riskScore: 7,
```

Comment:
I think not encrypted + may not require auth on the other end so should be > email? 10?